nxc/connection.py

```diff
@@ -437,6 +464,7 @@ def login(self):
                         owned[user_index] = True
                         if not self.args.continue_on_success:
                             return True
+            return None
```

Comment:
Why adding a none return? Isn’t that just boilerplate without any functionality?